central/notifier/service/service_impl.go

```diff
@@ -228,6 +228,8 @@ func (s *serviceImpl) TestUpdatedNotifier(ctx context.Context, request *v1.Updat
 	}()
 
 	if err := notifier.Test(ctx); err != nil {
+		log.Warnf("test notifier %q of type %q failed: %s: %v", request.GetNotifier().GetId(), request.GetNotifier().GetType(), err.Error(), err.Unwrap())
```

Comment:
How this line looks like?
Why do we need to log err and unwrap? Loggin err should be enough or we could use `+v` to get stacktrace.

Reply:
OK, got it `Error` will just print message and unwrap will give the underlying error